sagemaker-pipeline-parameterization/README.md

```diff
@@ -0,0 +1,11 @@
+# Parameterize SageMaker Pipelines
+
+Customers can use SageMaker Pipelines to build scalable machine learning pipelines that preprocess data and train machine learning models. With SageMaker Pipelines customers have a toolkit for every part of the machine learning lifecycle that provides deep customizations and tuning options to fit every organization. Customers have the freedom to customize SageMaker Pipelines to specific use cases, but also to create generic machine learning pipelines that can be reused across different use cases.
```

Comment:
Grammar nit - add comma for readability - `With SageMaker Pipelines, customers have...`